idseq_pipeline/commands/postprocess_functions.py

```diff
@@ -316,29 +317,33 @@ def run_stage3(lazy_run=False):
                         (SAMPLE_S3_INPUT_PATH, os.path.basename(local_file),
                          INPUT_DIR))
 
-    # generate taxid fasta
-    log_params = return_merged_dict(
-        DEFAULT_LOG_PARAMS,
-        {"title": "run_generate_taxid_fasta_from_hit_summaries"})
+    def default_log_param_plus_title(title):
+        return return_merged_dict(DEFAULT_LOG_PARAMS, {"title": title})
```

Comment:
since this is the only place DEFAULT_LOG_PARAMS is used, maybe we can get rid of the global and just put that hash in here?